content/en/account_management/rbac/permissions.md

```diff
@@ -306,7 +285,57 @@ This permission can be granted or revoked from a role via [the Roles API][1].
 
 ### Log Data Access
 
-The following permissions can be granted to manage read access on subsets of log data:
+Grant the following permissions to manage read access on subsets of log data:
+
+* `logs_read_data`(Recommended) It offers finer grained access control over log data thanks to restriction queries.
+* `logs_read_index_data` is the alternative approach to restrict data access to indexed log data on a per-index basis.
+
+Those two permissions are applied sequentially. Meaning that a user can be restricted to a subset of indexes and the restriction queries apply on top of it.
+
+**Example**: User A has access to index `audit` and index `errors` and is restricted to the query `service:api`
+When looking into the log explorer, this user only sees logs from the `service:api` into the `audit` and `errors` indexes.
+
+In addition, access to the Livetail can be restricted with the `logs_live_tail` permission regardless of the data access restriction of the user.
+
+
+#### logs_read_data
+
+Read access to log data. If granted, other restrictions then apply. Such as `logs_read_index_data` or with [restriction query][4].
+
+If a user belongs to multiple roles, permissions combine one with another. 
+
+**Example**:
+
+* If a user belongs to a Role with Log Read Data and also belongs to a Role without Log Read Data, then he has the permission to read data.
+* If a user has a restriction to `service:sandbox` through one Role, and has another restriction to `env:staging` through another Role, then that user can access all `env:staging` logs (even those that are not from `service:sandbox`. And conversely.
+
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+**Global access**:
+
+Go to your [Datadog Roles page][1] and select the checkbox `read` as below for the wanted role:
+
+{{< img src="account_management/rbac/logs_read_data_access.png" alt="Read Data Access"  style="width:70%;">}}
+
+**Subset of Data**:
+
+This configuration is only supported through API
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+Revoke or grant this permission from a role via [the Roles API][1]. 
+Use [Restriction Queries][2] to scope the permission to a subset of Log Data. 
+
+
+[1]: /api/#roles
+[2]: /api/?lang=bash#roles-restriction-queries-for-logs
```

Comment:
Note: This link will not work starting Monday as we're redirecting to the new API page. Do we have an ETA on the status of the spec file for this?